Pokedex_ios/ViewModel/AdvancedFilterPopupViewModel.swift

```diff
@@ -68,4 +72,39 @@ class AdvancedFilterPopupViewModel {
         }
         return filtered
     }
+
+    @discardableResult
+    func filterByHeights() -> [PokemonRaw] {
+        guard !selectedHeights.isEmpty else { return filtered }
+        var pokemonList: [PokemonRaw] = []
+        for height in selectedHeights {
+           switch height {
+           case .small:
+               pokemonList += self.filterSmall(pokemons: self.filtered)
+           case .medium:
+               pokemonList += self.filterMedium(pokemons: self.filtered)
+           case .large:
+               pokemonList += self.filterLarge(pokemons: self.filtered)
+           }
+        }
+        self.filtered = pokemonList.sorted(by: { $0.id < $1.id })
```

Comment:
is this sort part of your results? because the pokemon should be sorted on the sort options only

Reply:
Before rebase the develop code I was appliying this sort, but now I see your point.

Reply:
Removed sorting